.changeset/lazy-planes-reply.md

```diff
@@ -0,0 +1,5 @@
+---
+'grafana-github-datasource': minor
+---
+
+Enable PDC for github datasource
```

Comment:
`Add support for PDC` or something like that instead? We don't enable it per say.